plugin/entryBase.go

```diff
@@ -68,16 +69,18 @@ func (e *EntryBase) attributes() EntryAttributes {
 	return e.attr
 }
 
-func (e *EntryBase) slashReplacementChar() rune {
+// SlashReplacementChar returns the value set by SetSlashReplacementChar.
+func (e *EntryBase) SlashReplacementChar() rune {
 	return e.slashReplacementCh
 }
 
 func (e *EntryBase) id() string {
 	return e.washID
 }
 
-func (e *EntryBase) setID(id string) {
-	e.washID = id
+// SetParent sets the entry's ID based on its parent's ID. Used by plugincached.List.
+func (e *EntryBase) SetParent(parent Entry) {
```

Comment:
Nit: `setIDFromParent` is more reflective of what's going on here.

Reply:
Does that matter to the people calling it?

Reply:
Does that matter to the people calling it?

Reply:
Maybe for someone new to the codebase, but even then it's probably not a big deal. That's why I marked it as a nit.